modules/league.py

```diff
@@ -745,6 +723,10 @@ async def season(self, ctx, *, arg=None):
                     line = f'{team_str}`{str(standing[1]) + "W":.<3} {str(standing[2]) + "L":.<3} {str(standing[3]) + "I":.<3} - {str(standing[4]) + "W":.<3} {str(standing[5]) + "L":.<3} {standing[6]}I`'
                     output.append(line.replace(".", "\u200b "))
 
+            STANDINGS[pro_value, season] = output
+        else:
+            output = STANDINGS[pro_value, season]
+
         await utilities.buffered_send(destination=ctx, content='\n'.join(output))
```

Comment:
Is `buffered_send` still necessary with the games list option removed?

Reply:
Nope, removed.